test/testdata/fixtures/single-dc-dse/k8ssandra.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       - metadata:
           name: dc1
         k8sContext: kind-k8ssandra-0
-        size: 1
+        size: 2
```

Comment:
Why do we need this change?

Reply:
Because there's a restart of the Cassandra pod, which triggered [this bug](https://github.com/k8ssandra/k8ssandra-operator/pull/671).

Now that the said bug is fixed, I can revert the change to a single instance if you want.

Reply:
Thanks for the info. I'd be in favor of keeping 1 to spare unnecessary resources.

Reply:
ok, I'll rebase on top of `main` and revert back to a single instance.